dune-project

```diff
@@ -22,7 +22,7 @@
   (ocaml
    (>= 4.02.3))
   ocamlfind
-  (fmt (>= 0.8.5))
+  (fmt (and (>= 0.8.5) (< 0.8.10)))
```

Comment:
This is only required for CI, not for releases

Reply:
That's true, as noted in the commit message. I didn't want to pollute this port with a backport of #354 as well.